server/util.py

```diff
@@ -242,3 +250,63 @@ def send_email_new_business_membership(account, contact):
 def name_splitter(name) -> tuple:
     name_array = name.split(" ", 1) if name else ["", ""]
     return name_array[0], name_array[1]
+
+
+def amount_to_charge(form=None, amount=None, pay_fees=None, interval=None) -> Decimal:
+    """
+    Determine the amount to charge. This depends on whether the payer agreed
+    to pay fees or not. If they did then we add that to the amount charged.
+    Stripe charges 2.2% + $0.30 on single charges and 2.2% + 0.5% + $0.30
+    on recurring charges.
```

Comment:
**question:** I remember way back that moving to stripe subscriptions meant an extra charge. Is that what the .5% is?

Reply:
That's right. Now that you mention it, I think I'll need to add a way of telling donation_adder if the donation in question is new or legacy because our plan, up to now, was that legacy recurring donations should not be on the hook for this new .5%. Let me know if you think that's changed though @ashley-hebler

Reply:
that's correct! We should not increase those fees for legacy recurring donations. I'll mention this to Liam as a reminder for our January conversion process

Reply:
Alright, I've added some logic to enforce this.

Reply:
nice! I feel like that was the last missing piece in this so I'll go ahead and mark this as approved for me 🚀